MinVer/MinVer.targets

```diff
@@ -21,12 +21,16 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(DesignTimeBuild)' != 'true'">
+  <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(UsingMicrosoftNETSdk)' == 'true' And '$(DesignTimeBuild)' != 'true'">
     <PropertyGroup>
       <MinVerVersion Condition=" '$(MinVerBuildMetadata)' != '' ">$(MinVerVersion)+$(MinVerBuildMetadata)</MinVerVersion>
       <Version>$(MinVerVersion)</Version>
       <PackageVersion>$(MinVerVersion)</PackageVersion>
     </PropertyGroup>
   </Target>
 
+  <Target Name="MinVer_Warnings" BeforeTargets="MinVer" Condition="'$(DesignTimeBuild)' != 'true'">
+    <Message Condition="'$(UsingMicrosoftNETSdk)' != 'true'" Text="WARNING: MinVer only works for Microsoft.NET.Sdk projects." Importance="high" />

```

Comment:
Perhaps the Warning task would be better here? https://docs.microsoft.com/en-us/visualstudio/msbuild/warning-task?view=vs-2017

Reply:
👍 absolutely. I'll change it.

Reply:
I've amended the commit.